experiments/scripts/run-subj.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+# Setup the environment variables
+source ./setup-vars.sh
+
+# Runs commands for "Instructions to setup a subject for test prioritization" section.
+bash ./setup-prio.sh
+
+# Instructions to generate automatically-generated tests for a subject
+/home/user/dependent-tests-impact/experiments/generate-sootTestOutput-auto.sh
```

Comment:
You should be able to use $DT_ROOT here. In fact, it would be even better if you moved the generate-sootTestOutput-auto.sh to scripts/ and use $DT_SCRIPTS here instead.